utils/abrt-action-analyze-java.c

```diff
@@ -120,34 +120,34 @@ write_results_to_file(const analysis_result_t *res_begin, const analysis_result_
 static char *
 backtrace_from_fd(int fdin)
 {
-    return xmalloc_read(fdin, /*no size limit*/NULL);
+    return libreport_xmalloc_read(fdin, /*no size limit*/NULL);
 }
 
 static char *
 backtrace_from_file(const char *file_name)
 {
-    return xmalloc_xopen_read_close(file_name, /*no size limit*/NULL);
+    return libreport_xmalloc_xopen_read_close(file_name, /*no size limit*/NULL);
 }
 
 static char *
 work_out_list_of_remote_urls(struct sr_java_stacktrace *stacktrace)
 {
-    struct strbuf *remote_files_csv = strbuf_new();
+    struct strbuf *remote_files_csv = libreport_strbuf_new();
```

Comment:
Can you look into replacing the libreport `strbuf` with `GString`? It’s another instance of a reinvented wheel.